### PR TITLE
#1338 Found an error in documentation of taskwait (wrong module import)

### PR DESCRIPTION
**Pull request by LonelyCat124**
The documentation of how to import taskwait used the old `from psyclone.transformations` which is not how this is imported.

### CODE REVIEW

File: src/psyclone/psyir/transformations/omp_taskwait_trans.py [hunk 1]
```diff
@@ -67,7 +67,8 @@ class OMPTaskwaitTrans(Transformation):
     >>> psy = PSyFactory(api).create(invokeInfo)
     >>>
     >>> from psyclone.transformations import OMPParallelTrans, OMPSingleTrans
-    >>> from psyclone.transformations import OMPTaskloopTrans, OMPTaskwaitTrans
+    >>> from psyclone.transformations import OMPTaskloopTrans
+    >>> from psyclone.psyir.transformations import OMPTaskwaitTrans
     >>> singletrans = OMPSingleTrans()
```
Comment (arporter) [inline src/psyclone/psyir/transformations/omp_taskwait_trans.py:72]: This same mistake is also made in the docstring of OMPTaskLoopTrans. Please could you fix it there too.

**Review by arporter — CHANGES_REQUESTED** [review]
Thanks for fixing. In checking the rebuilt documentation I noticed that we have the same mistake in one other location so it would be good to fix that too.
Apart from that, I've brought the branch up-to-date with master.
Docs all build fine. Tests and examples are all OK with compilation.

File: src/psyclone/psyir/transformations/omp_taskwait_trans.py (hunk 1 above)
Comment (LonelyCat124) [reply src/psyclone/psyir/transformations/omp_taskwait_trans.py:72]: Done :)

**Review by arporter — APPROVED** [review]
Thanks for fixing the second one.

**Pull request merged**